discovery-provider/src/utils/redis_constants.py

```diff
@@ -11,7 +11,15 @@
 trending_playlists_last_completion_redis_key = "trending-playlists:last-completion"
 challenges_last_processed_event_redis_key = "challenges:last-processed-event"
 user_balances_refresh_last_completion_redis_key = "user_balances:last-completion"
-latest_sol_play_program_tx_key = "latest_sol_play_program_tx_key"
-latest_sol_play_db_tx_key = "latest_sol_play_db_tx_key"
 latest_legacy_play_db_key = "latest_legacy_play_db_key"
 index_eth_last_completion_redis_key = "index_eth:last-completion"
+
+# Solana latest program keys
+latest_sol_play_program_tx_key = "latest_sol_program_tx:play:confirmed"
+latest_sol_play_db_tx_key = "latest_sol_program_tx:play:db"
+
+latest_sol_rewards_manager_program_tx_key = "latest_sol_program_tx:rewards_manager:confirmed"
```

Comment:
so for clarity, on chain these transactions are not actually just 'confirmed' i think they are 'finalized' - confirmed is what returns super quickly, finalized is later

Reply:
I can just switch this to `chain` for more clarity to differentiate between `db`